Charts/eiger/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: eiger
+description: A Helm chart for an Odin Eiger deployment
+
+type: application
+version: 0.1.0
+appVersion: "1.16.0"
```

Comment:
Note that you override appVersion when publishing the chart so this is redundant